lua/nvim-tree/explorer/explore.lua

```diff
@@ -41,14 +41,18 @@ local function populate_children(handle, cwd, node, git_status, parent)
 
       ---@type uv.fs_stat.result|nil
       local stat = vim.loop.fs_stat(abs)
+
+      -- Type must come from fs_stat and not fs_scandir_next to maintain sshfs compatibility
+      local type = (vim.loop.fs_stat(abs) or {}).type
```

Comment:
```suggestion
      local type = stat and stat.type or nil
```

It looks like we're still calling fs_stat twice, which is a big performance hit.

I might be misunderstanding, but it looks like type should be nil on an fs_stat failure.

Reply:
You're right, that's an oversight by me.